src/app/login/login.component.html

```diff
@@ -1,39 +1,44 @@
 <div id="login_wrapper" class="login-pf">
-	<span id="badge">
+    <span id="badge">
```

Comment:
We are using 2 spaces as a standard for the UI. Can you please switch this back?